source/wp-content/themes/wporg-showcase-2022/src/site-post-template-wrap/style.scss

```
@@ -0,0 +1,22 @@
+
+.wp-block-wporg-site-post-template-wrap {
+	&:hover,
+	&:focus {
+		outline: none;
+
+		img {
+			border: 1px solid var(--wp--custom--link--color--text);
```

Could just set color here
```suggestion
			border-color: var(--wp--custom--link--color--text);
```